WordPress/WordPressTest/Models/WPAccountTests.swift

```diff
@@ -0,0 +1,36 @@
+import XCTest
+
+class WPAccountTests: XCTestCase
+{
+    private let accountEntityName = "Account"
+    private let oneHundredYearsInMilliseconds: NSTimeInterval = 1000 * 60 * 60 * 24 * 7 * 52 * 100
+
+    private var account: WPAccount!
+    
+    override func setUp() {
+        super.setUp()
+        
+        let context = TestContextManager.sharedInstance().mainContext
+        account = NSEntityDescription.insertNewObjectForEntityForName(accountEntityName, inManagedObjectContext: context) as! WPAccount
+    }
+    
+    override func tearDown() {
+        account = nil
+        
+        super.tearDown()
+    }
+    
+    func testWasCreatedBeforeHideViewAdminDateBeforeDate() {
+        let beforeDate = NSDate(timeIntervalSince1970: 0)
+        account.dateCreated = beforeDate
```

Comment:
As a follow up to the comment above, if you want to make your tests even more resilient, I'd consider creating a different account object on each test, so that you can be sure the different tests are not affecting each other.

Feel free to ignore this if you think it's overkill.